docs_guide/intro/new_doc_project.rst

```diff
@@ -6,18 +6,18 @@ Create a New Documentation Project
 
 .. include:: ../others/tutorial_type.rst
 
-In this tutorial you will use the ``sphinx-quickstart`` command to create a new documentation project for a Qiskit repository with Sphinx. Then, you will structure the resulting documentation according to
+In this tutorial you will use the :doc:`sphinx-quickstart <sphinx:man/sphinx-quickstart>` command to create a new documentation project for a Qiskit repository with :doc:`Sphinx <sphinx:index>`. Then, you will structure the resulting documentation according to
```

Comment:
```suggestion
In this tutorial you will use the :doc:`sphinx:man/sphinx-quickstart` command to create a new documentation project for a Qiskit repository with :doc:`Sphinx <sphinx:index>`. Then, you will structure the resulting documentation according to
```

I prefer directing linking without specifying the link title whenever possible to reduce maintenance.